css/drop.scss

```diff
@@ -0,0 +1,110 @@
+$containerMargin: 20px;
+
+#app-content-dropit {
+	display: flex;
```

Comment:
This apparently makes the dropit page be shown everywhere

Reply:
Fix pushed.